gcovr/tests/simple1-drive-subst/Makefile

```diff
@@ -0,0 +1,53 @@
+
+GCOVR_TEST_DRIVE ?= Z:
+
+override GCOVR += --verbose
+
+subst:
+ifeq ($(wildcard $(GCOVR_TEST_DRIVE)/*.*),)
+	cmd.exe /C "subst $(GCOVR_TEST_DRIVE) .." || exit 0
+endif
```

Comment:
What will happen if a Z:/ drive already exists but is used for a different purpose? Can the test destroy data? When we do similar stuff on Linux, we ensure that we're running in an isolated Docker container. That's not an option with Windows (AFAIK) but some level of safety might be desirable.

Perhaps the reasonable approach would be:

* do not hardcode a test drive in the Makefile
* if the Makefile is not given a test drive, show an error and fail the test
* in some part of the Python code (Noxfile? test_gcovr?) find a unused drive letter and pass it in a variable to the Makefile

Reply:
The subst will fail if there is already a drive connected. I dropped the idea of searching a free drive letter because if the test fails or is aborted or the option `--skip_clean` is used the substitution isn't removed and a new drive is used in the next call until all drive letters are in use.
I'll change the makefile to subst the drive immediately before and remove the drive after each call.

Reply:
Now it's working with the dynamic selection of a free drive.